apps/web/playwright/integrations-stripe.e2e.ts

```diff
@@ -110,14 +109,22 @@ test.describe("Stripe integration", () => {
     await expect(await page.locator('[data-testid="cancelled-headline"]').first()).toBeVisible();
   });
 
+  todo("should not send a booking paid event when the payment is not confirmed by Stripe");
+
```

Comment:
In my mind, this makes sense, but I could not make this assertion work.

IDK if we have a bug and we need to wait for provider confirmation to fire the webhook

Reply:
or maybe create a new webhook, `PAYMENT_CONFIRMED` or something like that